pkg/stripe/verbosetransport.go

```diff
@@ -0,0 +1,80 @@
+package stripe
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/stripe/stripe-cli/pkg/ansi"
+)
+
+// inspectHeaders is the whitelist of headers that will be printed.
+var inspectHeaders = []string{
```

Comment:
Good call on the whitelist (as opposed to blacklist) — these really don't change much.

May want to consider adding `Idempotent-Replayed` too.